src/pages/signing_official_console/DAACell.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { Checkbox } from '@mui/material';
+import { DAA } from '../../libs/ajax/DAA';
+import { Notifications } from '../../libs/utils';
+
+export default function DAACell(props) {
+  const {rowDac, researcher, institutionId, daas, refreshResearchers, setResearchers} = props;
+  const id = researcher && (researcher.userId || researcher.email);
```

Comment:
You can use the `?` syntax for several of these for clarity. `researcher?.userId`